bno.cpp

```diff
@@ -0,0 +1,31 @@
+#include "Bno.h"
+
+Bno::Bno()
+{
+    bnoSensor = Adafruit_BNO055(55);
+}
+
+//Obtain an int with the X value of the BNO055.
+void Bno::checkAngle()
+{
+    sensors_event_t event;
+    bnoSensor.getEvent(&event);
+    bnoSensor.angle = event.orientation.x;
+}
+
+//Check if the robot is on or off the ground.
+void Bno::checkLift()
+{
+    bool lift = 0;
+    sensors_event_t event;
+    bnoSensor.getEvent(&event);
+    int height = event.orientation.y;
+    if (height > 10)
+    {
+        bnoSensor.lift = true;
+    }
+    else
+    {
+        bnoSensor.lift = false;
+    }
+}
```

Comment:
bnoSensor is an instance of Adafruit_BNO055(55), lift is an attribute of your BNO class. Does Adafruit_BNO055 has a lift attribute to? Maybe this function could only return true or false, the lift attribute seems irrelevant.